### DISCUSSION TITLE
[1.0] Some machines/items lack icons in their slots

### DISCUSSION THREAD

**Discussion by Wheatley1001** [FakeFishGames/Barotrauma#11239] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Many items and machines were updated to include many incredibly useful icons to show new players what items go in what machine/item. There were some, however, which were excluded. Below is a list of all the items I found which do not have icons yet have specific items that belong to them.

![image](https://user-images.githubusercontent.com/105495756/218346441-d6a56c79-e40d-40f3-8aff-856e7418fd7c.png)
_Figure 1. Charging dock, item needed: battery cells, fulgurium battery cells._

![image](https://user-images.githubusercontent.com/105495756/218350656-7632bea9-692a-4ec2-b1db-24995a120bba.png)
_Figure 2. Nuclear Reactor, item needed: fuel rod._

![image](https://user-images.githubusercontent.com/105495756/218350691-edee5f46-eb0b-4f61-b7d3-35e56e759aa0.png)
_Figure 3. Diving suit locker, item needed: diving suit (any)._

![image](https://user-images.githubusercontent.com/105495756/218350766-436d5508-692f-4d3c-af8e-19b1ece92e65.png)
_Figure 4. Ammunition shelf, item needed: pulse gun ammo (any), coilgun ammo (any), flak gun ammo (any), depth charge (any)._

![image](https://user-images.githubusercontent.com/105495756/218350794-7331def8-ea45-4d56-a625-9eb36e00c6c4.png)
_Figure 5. Railgun shell rack, item needed: railgun shell._

![image](https://user-images.githubusercontent.com/105495756/218351042-13222978-0934-41cd-879d-d43a6d276990.png)
_Figure 6. Research station, items needed: genetic material, unidentified genetic material, stabilozine._

### Bug prevalence

Happens every time I play

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
![image](https://user-images.githubusercontent.com/73229309/219061411-aa85c2dc-e954-4d4e-911d-d742c4f027e0.png)

```xml
     <SlotIcon slotindex="0" texture="Content/Map/LabelIcons.png" sourcerect="512,0,256,256" origin="0.5,0.5" />
```
